doc/tutorials/content/sources/custom_executor/custom_executor.cpp

```diff
@@ -0,0 +1,81 @@
+#include <pcl/common/generate.h>
+#include <pcl/filters/functor_filter.h>
+#include <pcl/visualization/cloud_viewer.h>
+#include <chrono>
+
+namespace executor
+{
+// Forward declaration for custom executor
+template <typename Blocking, typename ProtoAllocator>
+struct omp_benchmark_executor;
```

Comment:
Is this forward declaration required? What happens if you move the `is_executor_available` after the class is defined?

Reply:
It is needed since we check for this inside the class:
```cpp
static_assert(is_executor_available_v<omp_executor>, "OpenMP benchmark executor unavailable");
```

Reply:
That static assert checks for `omp_executor`, not `omp_benchmark_executor`

Reply:
Oops. Fixed now.